data/transactions/logic/opcodes.go

```diff
@@ -530,38 +530,28 @@ var OpSpecs = []OpSpec{
 
 	{0x60, "balance", opBalance, proto("i:i"), 2, only(ModeApp)},
 	{0x60, "balance", opBalance, proto("a:i"), directRefEnabledVersion, only(ModeApp)},
-	{0x60, "balance", opBalance, proto("b:i"), sharedResourcesVersion, only(ModeApp)},
 	{0x61, "app_opted_in", opAppOptedIn, proto("ii:i"), 2, only(ModeApp)},
```

Comment:
question: I was reading const `sharedResourcesVersion = 9` and it is no longer used if we remove these previous lines. and my take from `README(_in).md` suggests that we are deprecating these opcodes that takes relative index in next version, with opcodes taking absolute index as substitution?

Reply:
We are removing this line because its purpose was to change the `balance` opcode to require a `bytes` arg instead of `any`. That was in the previous commit, because the thought was that we were removing the use of "slot" arguments for opcodes like this in v9. But since we came up with a way to keep them, `balance` should stay the same, and take `any`.

No final decision to deprecate the slot stuff in v10 has been made, but we're at least starting to encourage people not to use it.

Reply:
I see, this is really for the sake of banning low number ID that might lead to confusion to using slot IDs.  Maybe eventually we (might) want to remove slot ID usage, but for now the low number ID banning seems ok (and necessary).